### PR TITLE
Fix missing error dialog (DEV)

### PR DESCRIPTION
**Pull request by d4rken**
Didn't we miss it? The error dialog? :hugs:

### CODE REVIEW

**Review by BMItr — DISMISSED** [review]
:ok_hand:

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/bugreporting/BugReporter.kt [hunk 1]
```diff
@@ -4,9 +4,21 @@ import de.rki.coronawarnapp.util.di.AppInjector
 
 interface BugReporter {
     fun report(throwable: Throwable, tag: String? = null, info: String? = null)
+
+    companion object {
+        val isAUnitTest: Boolean by lazy {
```
[the remark this answers is not in the document]
Comment (d4rken) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/bugreporting/BugReporter.kt:9]: Then it's resolved for any autocompletion call though, e.g. when typing "is...", we would get `isAUnitTest` as suggestion,  whereas now its context relation is to the bug reporting code is clear. I'd prefer not to have such a nice/workaround piece of code be part of general suggestions. What do you think?

[the remark this answers is not in the document]
Comment (chiljamgossow) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/bugreporting/BugReporter.kt:9]: I understand where you are coming from. For me, when reading the source code, BugReporter.isAUnitTest would be confusing. However, it is not bug.

[the remark this answers is not in the document]
Comment (d4rken) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/bugreporting/BugReporter.kt:9]: Would you find it less confusing if we move it to `CWADebug.isAUnitTest`?

**Review by ralfgehrer — APPROVED** [review]

**Review by BMItr — APPROVED** [review]

**Pull request merged**